styles/variables-base.less

```diff
@@ -436,17 +436,21 @@
 
 // Keypad
 // ---------------------------------------
-@agate-keypad-button-height:        inherit;
-@agate-keypad-height:               inherit;
-@agate-keypad-input-border-bottom:  inherit;
-@agate-keypad-input-font-size:      inherit;
-@agate-keypad-keycontainer-width:   inherit;
-@agate-keypad-label-bottom:         inherit;
-@agate-keypad-label-font-size:      inherit;
-@agate-keypad-label-line-height:    inherit;
-@agate-keypad-text-font-size:       inherit;
-@agate-keypad-text-line-height:     inherit;
-@agate-keypad-width:                inherit;
+@agate-keypad-button-height:           inherit;
+@agate-keypad-button-disabled-opacity: 0.3;
```

Comment:
Hm. is there GUI guide for this?
how about 0.6?

Reply:
No, I just took it from Button. I'll change it to 0.6.